fern/pages/api-definition/fern-definition/websockets.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: WebSockets in the Fern Definition 
+description: Learn how to define WebSockets in the Fern Definition
+---
+
+In Fern, you can specify WebSockets in your API definition. The WebSockets will be included in both the generated SDKs and the API documentation. 
+
+## WebSocket definition
+Each WebSocket is defined in its own file, where it is described by the `channel` object. 
+
+### The channel object
+
+A `channel` is defined by the following fields:
+
+- `auth`: The authentication scheme for the WebSocket
+- `path`: The path of the WebSocket 
+- `headers` _(Optional)_: Any headers the WebSocket will send
+- `path-parameters` _(Optional)_: Any path parameters in the WebSocket path
+- `query-parameters` _(Optional)_: Any query parameters used in the initial request of the WebSocket
+- `messages` _(Optional)_: The schemas of the messages the WebSocket can send and receive once connected
+    - `origin`: The entity that sent the message (e.g. `client` or `server`)
+    - `body`: The schema of the message
+- `examples`: Example WebSocket connection _(Optional)_
+
+### WebSocket example 
+
+<CodeBlock title='chat.yml'>
+  ```yaml
+  channel:
+    path: /chat
+    auth: false
+    query-parameters:
+      model_id:
+        type: optional<string>
+        docs: The unique identifier of the model.
+      model_version:
+        type: optional<integer>
+        docs: The version number of the model.
+    messages:
+      publish:
+        origin: client
+        body: PublishEvent
+      subscribe:
+        origin: server
+        body: SubscribeEvent
+  types:
+    PublishEvent:
+      docs: The input from the user to send through the WebSocket. 
+      properties: 
+        text: 
+          type: string
+          docs: The user text to send into the conversation. 
+    SubscribeEvent:
+      docs: The response from the server sent through the WebSocket.
+      properties: 
+        id: 
+          type: string
+          docs: The id of the message. 
+        message:
+          type: string
+          docs: The message sent through the socket. 
+  ```
+</CodeBlock>
+
+## WebSocket playground
```

Comment:
In addition to the WS Playground, there's a WebSocket Reference. The page design here is novel and worth calling out explicitly. Specifically the `handshake` and `messages`: https://dev.hume.ai/reference/empathic-voice-interface-evi/chat/chat